quick_start.md

```diff
@@ -131,9 +129,19 @@ Click **Create**.
 
 ![Add User](console/configurations/imgs/control_add_user.png)
 
-## Access created resource
+## Enroll a device
 
 - Enroll a new device for the created user by acessing the enrollment link
   - The link is sent to the email configured for the user or by sharing the link from the user details
 
-- Access the configured resource
+- The enrollment process needs to be performed from the device to be enrolled
```

Comment:
@GMartinez-Sisti the 2x enrollment seems a bit confusing.
How about something like
```
The enrollment process is done directly from the created user's device.
```
Do we have any steps for this in this repo? If so, we could add a link to them.

Reply:
Looking at it again, it looks confusing indeed. Changing.